src/supertux/console.cpp

```diff
@@ -384,7 +385,7 @@ Console::autocomplete()
   // depending on number of hits, show matches or autocomplete
   if (cmds.empty())
   {
-    m_buffer.addLines("No known command starts with \"" + prefix + "\"");
+    m_buffer.addLines(_("No known command starts with \"") + prefix + _("\""));
```

Comment:
Console output is visible in the debug console and/or stdout. The feature isn't intended to be used by users but for debugging. Thanks in advance for Chinese debug information which no one in the team understands.